src/main/java/edu/uw/edm/edmzuulproxy/healthcheck/GwsHealthIndicator.java

```diff
@@ -0,0 +1,39 @@
+package edu.uw.edm.edmzuulproxy.healthcheck;
+
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.boot.actuate.health.HealthIndicator;
+import org.springframework.stereotype.Component;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
+
+import edu.uw.edm.gws.GroupsWebServiceClient;
+import edu.uw.edm.gws.model.GWSSearchType;
+import edu.uw.edm.gws.model.GroupReference;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Component
+@Slf4j
+public class GwsHealthIndicator implements HealthIndicator {
+
+    private GroupsWebServiceClient groupsWebServiceClient;
+
+    @Autowired
+    public GwsHealthIndicator(GroupsWebServiceClient groupsWebServiceClient) {
+        this.groupsWebServiceClient = groupsWebServiceClient;
+    }
+
+    @Override
+    public Health health() {
+        String username = "edmsci"; // hardcode app user name for simplicity
```

Comment:
Would you have the username supplied be the application.yml otherwise if we wanted to use a different user we'd have to rebuild the application

Reply:
I think this suggestion is important. In the interest of time, could you create a separate issue to follow up on the next sprint?

Reply:
I would just do the work, rather than making a 1 pt story. This is probably 15 minutes of work.

Reply:
I don't think it is really required here. The name here really does not matter. It still works as expected even if the name is an invalid or non-existent netID. I tested this locally with an invalid netID and the call succeeded with an empty list. I'd prefer keeping it simple by not introducing another configuration.

Reply:
I added additional comment.